ansible/roles/grafana/templates/grafana_nginx.conf.j2

```diff
@@ -0,0 +1,20 @@
+server {
+    server_name _;
+
+    listen 3000 ssl default_server;
+    listen [::]:3000 ssl default_server;
```

Comment:
Not really. The only issue I can see with putting it on 443 is that we need to do some more magic to get letsencrypt to work properly (take down grafana docker, start it back up again upon renewal).